website/frontend/src/hooks/useApi.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import { GearApi } from '@gear-js/api';
+import { initApi } from '../api/initApi';
+
+let api: GearApi | null = null;
+
+export const useApi = (): [GearApi | null, boolean] => {
+  const [gear, setGear] = useState<null | GearApi>(api || null);
```

Comment:
could we rename this var name? I think it is not contains meaning

Reply:
which one?